shishito/runtime/platform/node_webkit/control_test.py

```diff
@@ -0,0 +1,28 @@
+"""
+@summary: Common configuration functions supporting test execution.
+ Various startup and termination procedures, helper functions etc.
+ Not to be used for directly testing the system under test (must not contain Asserts etc.)
+"""
+
+from shishito.runtime.platform.shishito_control_test import ShishitoControlTest
+
+
+class ControlTest(ShishitoControlTest):
+    """ ControlTest for node-webkit platform """
+
+    def start_browser(self):
+        # call browser from proper environment
+        self.driver = self.test_environment.call_browser('')
```

Comment:
If you indeed don't need to pass any `config_section` to `call_browser()` method, you should not pass any parameter at all (not even empty string, that looks weird) & remove that parameter from `call_browser()` method declaration.